src/watchlists/Popup.jsx

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from 'react';
```

Comment:
let move this component into the common folder and just simply use it in the watchlist feature